module/VuFind/src/VuFind/RecordDriver/EDS.php

```diff
@@ -115,6 +115,18 @@ public function getItemsAbstract()
         return $abstract[0]['Data'] ?? '';
     }
 
+    /**
+     * Get the abstract and summary notes.
```

Comment:
I think this comment is slightly out of date:

```suggestion
     * Get the abstract notes.
```